fenix/app/src/main/java/org/mozilla/fenix/shopping/ui/ProductAnalysis.kt

```diff
@@ -56,6 +57,8 @@ import java.util.SortedMap
 private val combinedParentHorizontalPadding = 32.dp
 private val productRecommendationImageSize = 60.dp
 
+private const val PRODUCT_RECOMMENDATION_SETTLE_TIME = 1500
```

Comment:
Let's suffix this with `_MS`

Reply:
Think this was missed

Reply:
Looks like I only did the one settle time. thanks for catching this